src/main/electron/listener.cljs

```diff
@@ -153,31 +157,35 @@
   (js/window.apis.on "quickCapture"
                      (fn [args]
                        (let [{:keys [url title content]} (bean/->clj args)
-                             page (or (state/get-current-page)
-                                      (string/lower-case (date/journal-name)))
+                             insert-today? (get-in (state/get-config)
+                                                   [:quick-capture-options :insert-today]
+                                                   false)
+                             page (if (true? insert-today?) (string/lower-case (date/journal-name)) (state/get-current-page))
                              format (db/get-page-format page)
                              time (date/get-current-time)
                              text (or (and content (not-empty (string/trim content))) "")
-                             link (if (not-empty title) (config/link-format format title url) url)
+                             link (if (string/includes? url "www.youtube.com/watch") (str title " {{video " url "}}") (if (not-empty title) (config/link-format format title url) url))
                              template (get-in (state/get-config)
                                               [:quick-capture-templates :text]
                                               "**{time}** [[quick capture]]: {text} {url}")
                              content (-> template
                                          (string/replace "{time}" time)
                                          (string/replace "{url}" link)
                                          (string/replace "{text}" text))]
-                         (if (and (state/get-edit-block) (state/editing?))
+                         (if (and (state/get-edit-block) (not state/editing?)) ; changed to not so that block is created at the end of the page
                            (editor-handler/insert content)
                            (editor-handler/api-insert-new-block! content {:page page
-                                                                          :edit-block? false
+                                                                          :edit-block? true
```

Comment:
If user is not editing the page, insert content to the cursor; Otherwise insert to the target page and set it as editing?

Reply:
Sounds good! Sorry I am really new to Clojurescript. How do I going about making this change, please?

Reply:
Ok so I made the [change](https://github.com/logseq/logseq/pull/7391/commits/f70e9661eb1061f4f53c670e6317e2a563311ac0) which I think works best.

- If user is typing something, insert content to bottom of page but retain the cursor at where the user is typing.
- If user is not typing anything, insert content to bottom of page and don't change to editing.